Include/Ashkal/Ashkal/RenderShapeVisitor.hpp

```diff
@@ -1,7 +1,9 @@
 #ifndef ASHKAL_RENDER_SHAPE_VISITOR_HPP
 #define ASHKAL_RENDER_SHAPE_VISITOR_HPP
 #include <QWidget>
+#include "Ashkal/Ashkal/Camera.hpp"
 #include "Ashkal/Ashkal/Point.hpp"
+#include "Ashkal/Shapes/Stage.hpp"
```

Comment:
Alphabetical order.

Reply:
What changes do you mean regarding the zoom?